Source/WebCore/layout/formattingContexts/inline/text/TextUtil.cpp

```diff
@@ -698,8 +698,8 @@ char32_t TextUtil::lastBaseCharacterFromText(StringView string)
     if (!string.length())
         return 0;
 
-    for (size_t characterIndex = string.length() - 1; characterIndex--;) {
-        auto character = string.characterAt(characterIndex);
+    for (size_t characterIndex = string.length(); characterIndex > 0; --characterIndex) {
```

Comment:
This condition could just be characterIndex, because a size_t > 0 is equivalent to checking if it is non-null.